stardog/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "maintainer": "support@stardog.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.6.3",
+  "name": "stardog",
+  "short_description": "A Stardog data collector for DataDog.",
+  "guid": "1b32f0d4-49ef-40fb-aec3-365e4e7cd6ee",
+  "support": "contrib",
+  "supported_os": ["linux","mac_os"],
```

Comment:
Is this integrations supported on windows? If not, please note that in the `Compatibility` section of the README :)

Reply:
The integration should work on windows as well.  I'll fix that.